src/uid2LocalStorageManager.ts

```diff
@@ -0,0 +1,23 @@
+import { isValidIdentity, Uid2Identity } from "./Uid2Identity";
+
+export class UID2LocalStorageManager {
+  public setValue(identity: Uid2Identity) {
+    const value = JSON.stringify(identity);
+    localStorage.setItem("UID2-identity", value);
```

Comment:
Avoid magic strings. Make sure there's a single source of truth for `UID2-identity` - in some cases using a type to enforce the string being correct everywhere is an option, but I don't think we can do that here so a constant is probably the move.

Reply:
As I mentioned in another comment, we often end up with a few different UID2 values from different sources (i.e. the publisher storing their own things), so I think something like `UID2-sdk-identity` would make it easier to trouble-shoot integrations.

Reply:
defined and exporting the const from this file now - I'm using it in the mocks as well (although I don't have to)